src/DynamoCoreWpf/Interfaces/IWatchHandler.cs

```diff
@@ -220,7 +220,7 @@ private static string ToString(object obj)
         {
             return ReferenceEquals(obj, null)
                 ? Resources.NullString
-                : (obj is bool ? obj.ToString().ToLower() : obj.ToString());
+                : (obj is bool ? obj.ToString().ToLower() : obj is double? ((double)obj).ToString(CultureInfo.InvariantCulture) : obj.ToString());
```

Comment:
what happens if this is not done?

Reply:
This is the cause of the legacy issue I believe @jnealb mentioned that double value in watch node will display with custom user setting